paasta_tools/marathon_serviceinit.py

```diff
@@ -149,15 +151,22 @@ def status_marathon_job(service, instance, app_id, normal_instance_count, client
                                      running_instances, normal_instance_count)
 
 
-def get_verbose_status_of_marathon_app(app):
+def get_verbose_status_of_marathon_app(marathon_client, app, service, instance, cluster, soa_dir):
     """Takes a given marathon app object and returns the verbose details
     about the tasks, times, hosts, etc"""
     output = []
     create_datetime = datetime_from_utc_to_local(isodate.parse_datetime(app.version))
     output.append("  Marathon app ID: %s" % PaastaColors.bold(app.id))
     output.append("    App created: %s (%s)" % (str(create_datetime), humanize.naturaltime(create_datetime)))
-    output.append("    Tasks:")
 
+    autoscaling_info = get_autoscaling_info(marathon_client, service, instance, cluster, soa_dir)
```

Comment:
Does it not seem strange that we print this for every marathon app, but it should be "global" for an app/instance, right?

Also this is relatively expensive to run, and many don't have autoscaling info. (see line 375)